search-sentiments-map/routes/search.js

```diff
@@ -80,7 +196,9 @@ async function updateSearchResults() {
     // When testing ,use i < 1 to test for only one trend, and comment out
     // `await new Promise` line to avoid 1 minute pauses.
     for (let i = 0; i < trends.length; i++) {
-      await updateSearchResultsForTopic(trends[i].trendTopic);
+      updateSearchResultsForTopic(trends[i].trendTopic);
```

Comment:
Any reasoning for whether we need await here?

Reply:
I don't think it was necessary, that's why I deleted it.